scripting/pugsetup/captainpickmenus.sp

```diff
@@ -195,6 +195,17 @@ static int GetNextCaptain(int captain) {
       g_PickCounter--;
       return captain;
     }
+  if (GetConVarInt(g_SnakeCaptainsCvar) == 2) {
```

Comment:
I think this part is *not* supposed to be in the `GetConVarInt(g_SnakeCaptainsCvar) == 1)` section?

I think that's the source of the compilation errors: https://travis-ci.org/splewis/csgo-pug-setup/builds/593847907?utm_source=github_status&utm_medium=notification

Reply:
Yeah my bad... thats what I get for trying to do this in the github editor.